checkov/cloudformation/checks/resource/BaseCloudsplainingIAMCheck.py

```diff
@@ -12,28 +12,40 @@
 
 class BaseCloudsplainingIAMCheck(BaseResourceCheck):
     def __init__(self, name, id):
-        super().__init__(name=name, id=id, categories=CheckCategories.IAM, supported_resources=["AWS::IAM::Policy", "AWS::IAM::ManagedPolicy"])
+        super().__init__(name=name, id=id, categories=CheckCategories.IAM,
```

Comment:
@njgibbon Please note `categories` is plural, and supports multiple categories. Please wrap the `CheckCategories.IAM` in `[]` 🙏